shoop/core/models/_contacts.py

```diff
@@ -359,7 +359,7 @@ def get_person_contact(user):
         'last_name': getattr(user, 'last_name', ''),
         'email': getattr(user, 'email', ''),
     }
-    return PersonContact.objects.get_or_create(user=user, defaults=defaults)[0]
+    return PersonContact.objects.filter(is_active=True).get_or_create(user=user, defaults=defaults)[0]
```

Comment:
I think @tulimaki is correct. If user is logged in and admin deactivates this user, next page load should make sure user is active, if not, user will be logged out and messages.error or such should be added to notify user why this happened.

Returning `None` here might cause a lot of problems too.